tests/TestCase/Database/Type/JsonTypeTest.php

```diff
@@ -133,4 +134,19 @@ public function testToStatement()
     {
         $this->assertSame(PDO::PARAM_STR, $this->type->toStatement('', $this->driver));
     }
+
+    /**
+     * Test instance options
+     *
+     * @return void
+     */
+    public function testInstanceOptions()
+    {
+        // New instance to prevent others tests breaking
+        $instance = new JsonType();
+        $instance->setJsonOptions(JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES);
```

Comment:
I wonder if JSON_UNESCAPED_UNICODE should be the default.

Reply:
Maybe in 5.x